apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -36,41 +37,48 @@ const App = (): JSX.Element => {
 
 	return (
 		<>
-			<img alt="logo" src={reactLogo} width="30" />
+			<div className="header">
+				<img alt="logo" src={reactLogo} width="30" />
+				<ul className="header-nav">
+					<li>
+						<Link to={AppRoute.ROOT}>Root</Link>
+					</li>
+					<li>
+						<Link to={AppRoute.SIGN_IN}>Sign in</Link>
+					</li>
+					<li>
+						<Link to={AppRoute.SIGN_UP}>Sign up</Link>
+					</li>
+				</ul>
+			</div>
 
-			<ul>
-				<li>
-					<Link to={AppRoute.ROOT}>Root</Link>
-				</li>
-				<li>
-					<Link to={AppRoute.SIGN_IN}>Sign in</Link>
-				</li>
-				<li>
-					<Link to={AppRoute.SIGN_UP}>Sign up</Link>
-				</li>
-			</ul>
-			<p>Current path: {pathname}</p>
+			<main>
+				<Sidebar />
+				<div className="main">
```

Comment:
is it a good practice to use className="main" on a div inside a main tag? It doesn't affect anything, it just looks a bit  odd to me

Reply:
Yes, it should be named differently